### ISSUE TITLE
load checkpoint for stage1 of lam and only training stage2, but the loss becoming "Nan" after a few update

### ISSUE THREAD

**Issue by zhangyuan1994511** [OpenDriveLab/UniVLA#40]
Hi @retsuh-bqw 
    Thank you for sharing your excellent work! I have meet some problem about training lam, that is "load checkpoint for stage1 of lam and only training stage2, but the loss becoming "Nan" after a few update". Have you encountered the same problem before or could you offer some suggestions?
    I’d greatly appreciate any insights or suggestions you might have. Thank you for your time and help!

**Comment by zhangyuan1994511** [comment]
@retsuh-bqw 
The stage-2 training code the same as your released, and data same as your paper introduce
